### PR TITLE
A simple hack to provide global validate

### PR DESCRIPTION
**Pull request by woodensail**

### CODE REVIEW

**Comment by huei90** [comment]
Hi @woodensaid

Do you mind giving some information about this hack? Purpose? Use case? Fix what.

Thank you

File: dist/angular-validation.js [hunk 1]
```diff
@@ -665,6 +665,19 @@
...
+          $validationProvider.validators = [];
+          $validationProvider.validAll = function () {
+            return $validationProvider.validators.map(x=>x()).reduce((x,y)=>x&&y);
+          };
+        }
+        $validationProvider.validators.push(function () {
+          return checkValidation(scope, element, attrs, ctrl, validation, ctrl.$viewValue)
+        });
+
+        /**
```
Comment (huei90) [inline dist/angular-validation.js]: Try update the src/*.js file and use grunt to build dist folder
:+1:

Comment (woodensail) [inline dist/angular-validation.js]: Well, i'll try to learn the grunt.
BTW, it struck me that maybe i can use the validate group to achieve my purpose. I'll try it later.

Comment (woodensail) [inline dist/angular-validation.js]: I know where I was wrong, the input needs a name to reg into form but i missed it.
So i add nameAttr to every input. After that, the '$validation.validate($scope.myForm)' finally worked.
I'll close the PR.

**Comment by huei90** [comment]
Please reopen or open a new one if any suggestions :beers:

**Pull request closed without merging**